packages/openapi-fetch/tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "lib": ["ESNext", "DOM"],
```

Comment:
Fun fact: I originally wanted to add `allowJs: true` to TSConfig, however, that broke everything—it tries to infer types from `.js` files and ignores `.d.ts`. So fortunately no change is necessary, and TypeScript likes `.d.ts` just as much as the original `.ts`